wolfcrypt/src/ecc.c

```diff
@@ -2562,14 +2563,15 @@ static int ecc_mulmod(mp_int* k, ecc_point* tG, ecc_point* R, ecc_point** M,
                    /* ok window is filled so double as required and add  */
                    /* double first */
                    for (j = 0; j < WINSIZE; j++) {
-                       err = ecc_projective_dbl_point(R, R, a, modulus, mp);
+                       err = ecc_projective_dbl_point_safeg(R, R, a, modulus,
```

Comment:
Typo? This fails to compile with --disable-harden.

Reply:
Yep!
Picked up in the Jenkins tests and fixed.
Thanks!